types/joint.d.ts

```diff
@@ -3041,6 +3041,66 @@ export namespace layout {
 
         export function fromGraphLib(glGraph: any, opt?: { [key: string]: any }): dia.Graph;
     }
+
+    export namespace Port {
+
+        interface PortOptions {
```

Comment:
Just `Options` will do. It is inside of the `Port` namespace. The same for `PortLabelOptions`.